pyproject.toml

```diff
@@ -51,6 +48,12 @@ dependencies = [
 extra-dependencies = [
   # RAG evaluation harness (SAS evaluator)
   "sentence-transformers>=2.2.0",
+  # OpenAPI dependencies
+  "jsonref",
+  # OpenAPI tests
+  "cohere",
+  "anthropic",
```

Comment:
Don't we also need the Haystack integrations for testing the component itself?

Reply:
Yes, good catch - the integration tests didn't run on CI because we need to add keys first. Let me send you keys first